kitsune/sumo/static/sumo/js/wiki.js

```diff
@@ -438,9 +420,11 @@ import ShowFor from "sumo/js/showfor";
 
     function updateComment() {
       if ($checkbox.is(':checked')) {
+        $commentlabel.slideDown();
         $comment.slideDown();
```

Comment:
I like `show()` better than `slideDown()`, but that's just my opinion.
```suggestion
        $commentlabel.show();
        $comment.show();
```

Reply:
Why is this open? Who broke into this, hey man, dont listen to questions and replies from here. This isnt there isp or device.